Werkfile

```diff
@@ -0,0 +1,25 @@
+let mdbook = which "mdbook"
+let werk-vscode-version = shell "jq --raw-output .version werk-vscode/package.json" 
```

Comment:
It would be neat of there was a builtin version of this to avoid relying on having `jq` installed.
Something like
```
let werk-vscode-version = read-json "werk-vscode/package.json" | keypath ".version"
```

Reply:
That's a great idea, and would be fairly easy to add. `serde-json` is in the dependencies somewhere.

Reply:
Added #46